.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -303,14 +303,27 @@ jobs:
     - name: List excluded SYCL CTS categories
       if: inputs.tests_selector == 'cts' && env.CTS_TESTS_TO_BUILD == ''
       run: |
-        cat $PWD/devops/cts_exclude_filter
+        if [ "${{ contains(inputs.target_devices, 'opencl:cpu')  }}" = "true" ]; then
+          cat $PWD/devops/cts_exclude_filter_OCL_CPU
+        fi
+        if [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
+          cat $PWD/devops/cts_exclude_filter_L0_GPU
+        fi
```

Comment:
IMO, we should drop this test and print that information *after* we've selected the right file. I'm ok if that will be done in another PR though.